### ISSUE TITLE
Add possibility to read Civilopedia from the main menu

### ISSUE THREAD

**Issue by ghost** [yairm210/Unciv#5432]
**Is your feature request related to a problem? Please describe.**
It would be great if it will be possible to open and read Civilopedia from the main menu. At the moment it is possible only after loading or starting any game.

**Describe the solution you'd like**
Add the little round button to the lower right corner of the main menu with big "C". Or just little standard button with "Civilopedia" text.

**Comment by SomeTroglodyte** [comment]
Has been considered, but so far not deemed important. Problems: UI is nice with 8 buttons, a 9th would disrupt the 2 column layout... And returning from the pedia, as screens are entirely discarded when navigating to a new one, and there is no screen history. Would need some crutch like new game does... As for UncivGame.Current.gameInfo being uninitialised at that point, some work already has been done, but would need more testing.

**Comment by SomeTroglodyte** [comment]
Now with easier access to varying base rulesets the additional question arises - which ruleset should a 'pedia called from main menu display? Before, just using the built-in vanilla was OK, now it's not so clear anymore. Better choose a ruleset first using the existing method (namely starting a game). I vote 'wontfix'.

**Comment by logicminimal** [comment]
> Now with easier access to varying base rulesets the additional question arises - which ruleset should a 'pedia called from main menu display? Before, just using the built-in vanilla was OK, now it's not so clear anymore. Better choose a ruleset first using the existing method (namely starting a game). I vote 'wontfix'.

just add a button to change the rulesets in the main menu

**Comment by AdityaMH** [comment]
#5856

**Comment by AdityaMH** [comment]
Civilopedia "each mod (option button)" in main menu?
